src/Components/SoundWave/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { useRef, useEffect, useState } from "react";
+import PropTypes from "prop-types";
+
+function SoundWave({ src, waveColor, progressColor }) {
```

Comment:
Could we give `src` another name to make it more clear what it is. Maybe like `audioFile`?

- [ ] Change `src` variable name.